darwin/future/data_objects/properties.py

```diff
@@ -81,31 +81,39 @@ class FullProperty(DefaultDarwin):
     team_id: Optional[int] = None
     annotation_class_id: Optional[int] = None
     property_values: Optional[List[PropertyValue]] = None
+    granularity: PropertyGranularity
+    dataset_ids: Optional[List[int]] = None
     options: Optional[List[PropertyValue]] = None
     granularity: PropertyGranularity = PropertyGranularity("section")
 
     def to_create_endpoint(
         self,
     ) -> dict:
-        if self.annotation_class_id is None:
+        if (
+            self.annotation_class_id is None
+            and self.granularity != PropertyGranularity.item
```

Comment:
double checking this now so could maybe have

```
    if self.granularity != PropertyGranularity.item:
        if self.annotation_class_id is None:
            raise ValueError("annotation_class_id must be set")
        include_fields["annotation_class_id"] = True
```
after `include_fields = ` so is clearer

Reply:
Nice suggestion, this is cleaner